src/data/reducers/productDataReducer.jsx

```diff
@@ -0,0 +1,24 @@
+import { GET_PRODUCT_DATA, GET_PRODUCT_DATA_SUCCESS } from '../types/types';
+
+const initialState = {
+  productData: {},
+};
+
+const productDataReducer = (state = initialState, action) => {
```

Comment:
For future reference, redux toolkit does reducers a bit differently, see: https://redux-toolkit.js.org/api/createReducer